spec/index.bs

```diff
@@ -302,12 +306,21 @@ and disable WebGPU on drivers with known bugs that can't be worked around.
 
 ### Timing attacks ### {#security-timing}
 
-WebGPU is designed for multi-threaded use via Web Workers. Some of the objects,
-like {{GPUBuffer}}, have shared state which can be simultaneously accessed.
-This allows race conditions to occur, similar to those of accessing a SharedArrayBuffer
-from multiple Web Workers, which makes the thread scheduling observable
-and allows the creation of high-precision timers.
-The theoretical attack vectors are a subset of those of SharedArrayBuffer.
+WebGPU is designed for multi-threaded use via Web Workers. As such, it is designed not to open
+the users to modern high-precision timing attacks. Some of the objects, like `GPUBuffer`,
+have shared state which can be simultaneously accessed. This allows race conditions to occur,
+similar to those of accessing a `SharedArrayBuffer` from multiple Web Workers,
+which makes the thread scheduling observable.
+
+The set of mitigations to address this attack vector, employed by user agents,
+can be a subset of [mitigations](https://hacks.mozilla.org/2020/07/safely-reviving-shared-memory/)
+against the malicious `SharedArrayBuffer` use. In particular,
+the user agent may limit sharing a `GPUBuffer` to Web Workers belonging to the same
+[browser context group](https://html.spec.whatwg.org/multipage/browsers.html#browsing-context-group).
```

Comment:
nit: brows**ing** context group

nit: Should be possible to make this a `[=browsing context group=]`, take a look at the `<pre class='anchors'>` at the top of the file